rwkv/sampling.py

```diff
@@ -18,9 +18,10 @@ def sample_probs(probs: np.ndarray, temperature: float = 1.0, top_p: float = 0.8
     if logit_bias is not None:
         logits = np.log(probs)
 
-        for token in logit_bias.keys():
-            logits[token] += logit_bias[token]
-
+        if ids := list(logit_bias.keys()):
+            logits[ids] += logit_bias[ids]
```

Comment:
This line gives me `TypeError: unhashable type: 'list'`, on Python 3.10. Probably, because `logit_bias` is a `Dict`, not a numpy array. I propose to either remove this optimization, or fix it for dicts.